keps/sig-network/4444-service-routing-preference/README.md

```diff
@@ -339,34 +295,37 @@ NOTE: Implementations reserve the right to refine the behavior associated with
 
 ### Notes/Constraints/Caveats (Optional)
 
-<!--
-What are the caveats to the proposal?
-What are some important details that didn't come across above?
-Go in to as much detail as necessary here.
-This might be a good place to talk about core concepts and how they relate.
--->
+None.
 
 ### Risks and Mitigations
 
-<!--
-What are the risks of this proposal, and how do we mitigate? Think broadly.
-For example, consider both security and how this will impact the larger
-Kubernetes ecosystem.
+* **Risk:** Having a routing preference like `PreferZone` comes at the risk of
+  endpoints in certain zones being overloaded if the originating traffic is
+  skewed towards a particular zone.
 
-How will security be reviewed, and by whom?
+  **Mitigation:**
+    * Emphasize in the documentation that the `PreferZone` preference is
+      designed for low-latency or monetory-cost reasons, with the understanding
+      that it can lead to overload within zones.
+    * Recommend approaches like having deployments per zone which can scale
+      independently of other zones.
 
-How will UX be reviewed, and by whom?
+* **Risk:** Users migrating from the `service.kubernetes.io/topology-mode`
+  annotation might encounter differences in exact routing behavior:
+  * The new field doesn't support a routing preference that is exactly similar
+    to using `service.kubernetes.io/topology-mode=Auto` from the old annotation.
+  * If both field and the annotation are set, the annotation will take precedence.
```

Comment:
...for some number of releases, until it is deprecated and removed

Reply:
Added